servers/lan-world.md

```
@@ -0,0 +1,96 @@
+
+1. Everyone closes the game and the launcher, but keeps Hamachi running (assuming you use Hamachi or another virtual LAN).
+2. Open **Windows PowerShell** as administrator and run:
+
+    ```ps
+    Get-NetIPInterface
+    ```
+
+<details>
+<summary>You'll see something like this</summary>
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)